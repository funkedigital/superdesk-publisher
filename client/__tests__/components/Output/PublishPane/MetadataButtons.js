import React from "react";
import MetadataButtons from "../../../../components/Output/PublishPane/MetadataButtons";
import { render } from "@testing-library/react";

describe("Output/PublishPane/MetadataButtons", () => {
  it("renders properly", async () => {
    const { container } = render(
      <MetadataButtons
        open={jest.fn()}
      />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
