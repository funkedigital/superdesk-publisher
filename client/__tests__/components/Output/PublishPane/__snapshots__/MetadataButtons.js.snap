// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output/PublishPane/MetadataButtons renders properly 1`] = `
<div
  class="toggle-box toggle-box--dark sp--dark-ui"
>
  <div
    class="toggle-box__header"
  >
    <div
      class="toggle-box__chevron"
    >
      <i
        class="icon-chevron-right-thin"
      />
    </div>
    <div
      class="toggle-box__label"
    >
      Meta data
    </div>
    <div
      class="toggle-box__line"
    />
  </div>
  <div
    class="toggle-box__content-wraper"
  >
    <div
      class="toggle-box__content"
    >
      <div
        class="sd-list-item-group sd-shadow--z1 no-margin"
      >
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                Facebook
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                Twitter
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                SEO / Meta Tags
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
